partcad/tests/unit/test_context.py

```diff
@@ -12,7 +12,7 @@
 
 
 def test_ctx1():
-    ctx = pc.Context("partcad/tests")
+    ctx = pc.Context("partcad/tests/partcad-examples.yaml")
```

Comment:
Unrelated change and it's not equivalent: another configuration file gets used. Let's discuss it and track as a separate issue/project.

Reply:
I will remove these kind of changes.

Reply:
I will remove these kind of changes.